os/arch/arm/src/s5j/Make.defs

```diff
@@ -117,9 +117,13 @@ endif
 CHIP_CSRCS += s5j_boot.c s5j_irq.c
 CHIP_CSRCS += s5j_serial.c
 CHIP_CSRCS += s5j_mac.c
-CHIP_CSRCS += s5j_watchdog.c
 CHIP_CSRCS += s5j_systemreset.c
 
+CHIP_CSRCS += s5j_watchdog.c
+ifeq ($(CONFIG_WATCHDOG),y)
```

Comment:
CONFIG_S5J_WATCHDOG is correct. it will fix build error
@btheosam benix, we tested without watch dog in local we need it ?
I checked it works properly without watchdog.
please verify it and let us know.

Reply:
ifeq ($(CONFIG_S5J_WATCHDOG),y)
CHIP_CSRCS += s5j_watchdog.c
ifeq ($(CONFIG_WATCHDOG),y)
CHIP_CSRCS += s5j_watchdog_lowerhalf.c
endif
endif

Reply:
in config, we modified that select CONFIG_WATCHDOG if user select CONFIG_S5J_WATCHDOG
In this case we don't have to use CONFIG_WATCHDOG because it under the CONFIG_S5J_WATCHDOG